src/codewhisperer/util/editorContext.ts

```diff
@@ -94,10 +93,9 @@ export async function buildListRecommendationRequest(
     // (1) User is CrossFile user group
     // (2) The supplemental context is from Supplemental Context but not UTG(unit test generator)
     const isUtg = await isTestFile(editor, DependencyGraphFactory.getDependencyGraph(editor.document.languageId))
-    const supplementalContexts: CodeWhispererSupplementalContext | undefined =
-        CodeWhispererUserGroupSettings.getUserGroup() === CodeWhispererConstants.UserGroup.CrossFile && !isUtg
```

Comment:
@leigaol the real logic using `usergroup` is here

Reply:
hence it's removed

Reply:
OK. So regardless of the user group, cross file context is enabled. Correct?

Reply:
yes it's enabled regardless the usergorup